android-core/src/main/kotlin/com/mparticle/internal/MPLocationListener.kt

```diff
@@ -0,0 +1,21 @@
+package com.mparticle.internal
+
+import android.location.Location
+import android.location.LocationListener
+import android.os.Bundle
+import com.mparticle.MParticle
+
+class MPLocationListener(private val mParticle: MParticle?) : LocationListener {
```

Comment:
Is there a need for `mParticle` to be nullable here? Won't we always have an instance to pass in?